content/blog/production-ready-object-detection-model-training-workflow-with-hpe-machine-learning-development-environment.md

```diff
@@ -0,0 +1,1099 @@
+model = build_frcnn_model2(num_classes=61)
+ckpt = torch.load('model_8.pth',map_location='cpu')
+model.load_state_dict(ckpt['model'])
+_=model.eval()
+```
+
+```python
+_=predict(model,images_t_list,targets_t_list)
+```
+
+In the next part of our blog, we scale our model training using distributed training within HPE Machine Learning Development Environment & System. 
+
+# Part 4: Training on HPE Machine Learning Development & System
+
+[HPE Machine Learning Development Environment](https://www.hpe.com/us/en/solutions/artificial-intelligence/machine-learning-development-environment.html) is a training platform software that reduces complexity for ML researchers and helps research teams collaborate. HPE combines this incredibly powerful training platform with best-of-breed hardware and interconnect in [HPE Machine Learning Development System](https://www.hpe.com/us/en/hpe-machine-learning-development-system.html), an AI turnkey solution that we'll be using for the duration of the tutorial. 
+
+This notebook walks you the commands to run the same training as Step 3, but using the HPE Machine Learning Development Environment together with the PyTorchTrial API.
+All the code is configured to run out of the box. The main change is defining a `class ObjectDetectionTrial(PyTorchTrial)` to incorporate the model, optimizer, dataset, and other training loop essentials.
+You can view implementation details looking at `determined_files/model_def.py`
+
+We will show you how to: 
+
+* Run a distributed training experiment
+* Run a distributed hyperparameter search
+
+*Note: This notebook was tested on a deployed HPE Machine Learning Development System cluster, running HPE Machine Learning Development Environment (0.21.2-dev0).*
+
+Let's get started!
+
+- - -
+
+## Pre-req: Run startup-hook.sh
+
+This script will install some python dependencies, and install dataset labels needed when loading the Xview dataset:
+
+```bash
+## Temporary disable for Grenoble Demo
+wget "https://determined-ai-xview-coco-dataset.s3.us-west-2.amazonaws.com/train_sliced_no_neg/train_300_02.json"
+mkdir /tmp/train_sliced_no_neg/
+mv train_300_02.json /tmp/train_sliced_no_neg/train_300_02.json 
+wget "https://determined-ai-xview-coco-dataset.s3.us-west-2.amazonaws.com/val_sliced_no_neg/val_300_02.json"
+mkdir /tmp/val_sliced_no_neg
+mv val_300_02.json /tmp/val_sliced_no_neg/val_300_02.json
+```
+
+*Note that completing this tutorial requires you to upload your dataset from Step 2 into a publically accessible S3 bucket. This will enable for a large scale distributed experiment to have access to the dataset without installing the dataset on device. View [Determined Documentation](<* https://docs.determined.ai/latest/training/load-model-data.html#streaming-from-object-storage>) and [AWS instructions](<* https://codingsight.com/upload-files-to-aws-s3-with-the-aws-cli/>) to learn how to upload your dataset to an S3 bucket. Review the* `S3Backend` class in `data.py`
```

Comment:
Hi Alex, Please remove the "<*" and ">" characters when you hotlink the "Determined Documentation" and "AWS instructions"